File: Algorithms/Searching/LinearSearch/linearsearch.py
def linearsearch(array, n):
    for index, x in enumerate(array):
        if x == n:
            return index, x
        elif x > n:
            break
    return -1

# Test the function
print(linearsearch([3, 5, 9, 10, 11, 16], 6))

File: Algorithms/Searching/LinearSearch/linearsearch.js
function linearsearch(array, n) {
  for (let i = 0; i < array.length; i++) {
    if (array[i] == n) {
      return i;
    }
  }
  return -1;
}

// Test the function
let result = linearsearch([1, 2, 3, 4, 5, 6, 7, 8, 9], 7);
console.log(result);
